sdk/identity/azure-identity/setup.py

```diff
@@ -73,8 +73,8 @@
     install_requires=[
         "azure-core<2.0.0,>=1.0.0",
         "cryptography>=2.1.4",
-        "msal<1.0.0,>=0.4.1",
-        "msal-extensions~=0.1.2",
+        "msal<2.0.0,>=1.0.0",
```

Comment:
If there is no breaking changes, why not make it `msal<2.0.0,>=0.4.1` ? (open question)

Reply:
Complementary question: are we exposing anything from msal to customers? Even thourgh callback or something?

Reply:
msal 0.4.1 should work but I prefer not to declare support for preview versions now that we've all moved to GA.

Nothing of msal is exposed by design, i.e. any msal in our public API is a bug 😁

Reply:
I'm good with setting min version to 1.0.0.